### ISSUE TITLE
question about test result

### ISSUE THREAD

**Issue by yilonghe** [Media-Smart/vedatad#15]
![image](https://user-images.githubusercontent.com/38379457/145673444-e77cb80e-2293-4fe1-938b-a1910043b0af.png)
Hi,

I ran the code of test, and the results are shown as up.
The result I got is much lower than the result of the paper, what is the possible reason？

**Comment by gauthsvenkat** [comment]
@yilonghe Did you manage to find out the reason? My team and I are also running the same experiment and our results are also much lower than the results from the paper. 

cc - @reubenj @abel-vs @ljfhendriks

**Comment by ljl02521** [comment]
@yilonghe @gauthsvenkat Have you found the reason for the abnormal results?

**Comment by gauthsvenkat** [comment]
> @yilonghe @gauthsvenkat Have you found the reason for the abnormal results?

Not really. We had to do this as part of a project for a course but haven't really touched this after we were done with the course.
